Gopkg.toml

```diff
@@ -34,7 +34,9 @@
   name = "k8s.io/client-go"
   version = "kubernetes-1.10.4"
 
-
+[[constraint]]
+  name = "k8s.io/apimachinery"
+  version = "kubernetes-1.10.4"
```

Comment:
Just wondering why do we need `k8s.io/apimachinery` here if we just add a `flag....` command?

Reply:
You are totally right :) this comes from the initial approaches. Removed.